src/components/notes/list.jsx

```diff
@@ -1,116 +1,75 @@
-  const formatedUpdatedAt = updatedAt.toLocaleDateString(undefined, options)
-  return (
-    <tr className="c-table-row">
-      <th className="c-table-cell c-table-cell--primary">
-        <Item {...props} />
-      </th>
-      <td className="c-table-cell">
-        <time datatime={props.note.cozyMetadata.updatedAt}>
-          {formatedUpdatedAt}
-        </time>
-      </td>
-      <td className="c-table-cell">—</td>
-      <td className="c-table-cell">
-        <Button
-          theme="action"
-          extension="narrow"
-          icon={<Icon icon="dots" color="coolGrey" width="17" height="17" />}
-          iconOnly
-          label="actions"
-        />
-      </td>
-    </tr>
-  )
-}
-
-const List = props => {
-  const { notes } = props
-  return !notes || !notes.length ? null : (
-    <table className="notes-list c-table">
-      <thead className="c-table-head">
-        <tr className="c-table-row-head">
-          <th className="c-table-header">Nom</th>
-          <th className="c-table-header">Dernière mise à jour</th>
-          <th className="c-table-header">Partages</th>
-          <th className="c-table-header" />
-        </tr>
-      </thead>
-      <tbody>
-        {notes.map(note => (
-          <Row key={note._id} note={note} />
-        ))}
-      </tbody>
-    </table>
-  )
-}
-
-const ConnectedList = props => {
-  const { data, fetchStatus } = props.notes
-  // cozy-client statuses
-  const isLoading = fetchStatus === 'loading' || fetchStatus === 'pending'
-  return (
-    <div className="notes notes-list-container">
-      {isLoading ? (
-        <Spinner size="xxlarge" middle />
-      ) : (
-        <div>
-          <ListHeader />
-          <List notes={data} />
-        </div>
-      )}
-    </div>
-  )
-}
-
-*/
-
-const ListHeader = () => {
-  return (
-    <>
-      <HeaderMenu
-        left={<MainTitle tag="h1">Mes notes</MainTitle>}
-        right={<Add />}
-      />
-    </>
-  )
-}
-
-export default ListHeader
+))
+/* const cellStyles = { flexGrow: 1 }
+const nameCellStyles = { flexGrow: 10 } */
+const List = translate()(({ t }) => (
+  <Table>
+    <TableHead>
+      <TableRow>
+        <TableHeader className="tableCellName">
+          {t('Notes.List.name')}
+        </TableHeader>
+        <TableHeader className="tableCell">
+          {t('Notes.List.updated_at')}
+        </TableHeader>
+        <TableHeader className="tableCell">
+          {t('Notes.List.location')}
+        </TableHeader>
+        <TableHeader className="tableCell">
+          {t('Notes.List.sharings')}
+        </TableHeader>
+      </TableRow>
+    </TableHead>
+    <TableBody>
+      <TableRow className="tableRowEmpty">
+        <TableCell className="tableCellEmpty">
+          <EmptyComponent />
+        </TableCell>
+      </TableRow>
+    </TableBody>
+  </Table>
+))
+
+const TitleApp = translate()(({ t }) => (
+  <MainTitle className="u-pl-1">{t('Notes.List.my_notes')}</MainTitle>
```

Comment:
Why do we need the left padding here?

Reply:
Because we want to have the title aligned with column's name and I don't know how to do other way? 
<img width="101" alt="Capture d’écran 2019-12-18 à 11 58 33" src="https://user-images.githubusercontent.com/1107936/71080592-c0faf900-218d-11ea-9e46-4620ee6652cb.png">

Reply:
That's a correct way to do it imo. It's just that the relationship between the 2 is not explicit, maybe a comment would be good, but it's a nit.

Reply:
Let's see: 

I'll create: 
var PADDING_TABLE_FIRST_COLUMN = 1rem

and use it in a class. It'll be clearer :)